apps/omg/lib/omg/state.ex

```diff
@@ -33,12 +33,14 @@ defmodule OMG.State do
 
   require Utxo
 
-  @type exec_error :: Validator.exec_error()
+  @type exec_error :: Validator.can_process_tx_error()
+
+  @fallback_fee_claimer_address Application.fetch_env!(:omg, :fee_claimer_address)
```

Comment:
Yes, it is. The other place setting this config is release task run only for ch-ch. So for other parties this value is irrelevant (Watcher, perftest, tests) it fallbacks to this

Reply:
I don't think this is as safe as it should be. If someone omits the parameter from the supervisor, nothing will prevent this from working with the default parameter. 
Omg state should prevent booting if the parameter in child chain isn't there.

Reply:
Would the child chain tests protect us from starting OMG.State without a correct fee claimer address?

Reply:
I think I see your concern here. I can do this config value mandatory for the ch-ch in the release task. What I cannot see is how it affect running ch-ch in other scenarios like integration tests or running bare-metal / docker (in case the fallback option isn't possible).

I'm opting for creating an issue to provide this config better way. WDYT @InoMurko :question:

Reply:
Maybe implementing fallbacks is a trap we shouldn't be putting them in place.
I would rather see us updating the tests so that the correct value is passed.
I look at this as a VERY important setting. If this isn't set correctly - our software would continue working fine (there's no specific monitoring for the claimer fee address) but you'd be in production without generating any revenue...